### PR TITLE
feat (Web vault): Bump webvault to 2023.1.0

### PR DESCRIPTION
**Pull request by GeekCornerGH**
This pull request bumps web vault to 2023.1.0. I also changed link on the 404 page.
You'll probably need to implement avatar color changer.
Finally, you'll need to merge dani-garcia/vaultwarden#3132 to have 2fa.directory working.

### CODE REVIEW

**Comment by BlackDex** [comment]
Thx @GeekCornerGH, But i'm not seeing any difference between this PR and #106 . Where #106 has a fix implemented for the forwarded mail alias services, which yours is missing.

Also, what do you mean with avatar color changer? I'm probably missing something 😄

**Comment by GeekCornerGH** [comment]
Hi @BlackDex
I fixed link in 404 page [(here](url) The fix is [here](https://github.com/dani-garcia/bw_web_builds/pull/108/files#diff-6cefba2df9a3c8857fe3b5b29d997eaf7cd665dab69af4796a7000ac47776894R363-R367)

**Comment by BlackDex** [comment]
> Hi @BlackDex I fixed link in 404 page [(here](url) The fix is [here](https://github.com/dani-garcia/bw_web_builds/pull/108/files#diff-6cefba2df9a3c8857fe3b5b29d997eaf7cd665dab69af4796a7000ac47776894R363-R367)

That URL only has a `/` removed at the end compared to the PR #106, which doesn't matter as far as i can tell.
And the Forwarded Mail fix you point to is not valid anymore. Those CSS lines will not work because Bitwarden changed the way it looks. They changed it from a radio-list to a dropdown.

**Review by stefan0xC — COMMENTED** [review]
@BlackDex I think these are the differences to my PR. According to the chat in #vaultwarden:matrix.org I think there were probably more changes planned?

@GeekCornerGH I think it would not be necessary to add me as a Co-Author in this case (especially if you did the work of fixing the not-applied patches and then recreating the patch file independently of my PR).

**Comment by BlackDex** [comment]
I'm not seeing any changed which differ to #106. While i like the quick PR for this from both of you.
We can't merge both PR if they are almost the same, and this one is lacking a correct fix.

File: patches/v2023.1.0.patch [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+       </div>
+@@ -42,11 +42,10 @@
+         </a>
+       </p>
+       <p>
+-        You can <a href="/">return to the web vault</a>, check our
+-        <a href="https://status.bitwarden.com/">status page</a> or
+-        <a href="https://bitwarden.com/contact/">contact us</a>.
++        You can <a href="/">return to the web vault</a>, or
++        <a href="https://github.com/dani-garcia/vaultwarden/">contact us</a>.
```
[the remark this answers is not in the document]
Comment (GeekCornerGH) [reply patches/v2023.1.0.patch:22]: I don't think there is

**Comment by GeekCornerGH** [comment]
@BlackDex just noticed what you said

**Comment by GeekCornerGH** [comment]
@BlackDex @dani-garcia Just merge @stefan0xC pr, I'll add my changed on another pr when you'll merge it

**Pull request closed without merging**